docs/pages/desktop-access/getting-started.mdx

```diff
@@ -1,214 +1,179 @@
 ---
-title: Getting Started with Desktop Access
-description: Automatically add Teleport Desktop Access to your infrastructure
-videoBanner: v36tlB8D5kk
+title: Getting Started with Windows Access
```

Comment:
```suggestion
title: Getting Started with Desktop Access
```

The pages still appear under a `desktop-access` URL and the menu bar on the left still says "Desktop Access." Let's just stick with desktop access for now.